gradle/wrapper/gradle-wrapper.properties

```
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-6.7-rc-5-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-6.7-bin.zip
```

nit, you probably need to use -all:
```suggestion
distributionUrl=https\://services.gradle.org/distributions/gradle-6.7-all.zip
```
I don't know why cmd updater switches to -bin for 6.7..

---

thanks!